### PR TITLE
Drop support for SLES 12 repos

### PR DESCRIPTION
**Pull request by mjura**

### CODE REVIEW

**Comment by tboerger** [comment]
:+1:

**Comment by toabctl** [comment]
+1 (but that needs more changes in crowbar-core, right?)

**Comment by vuntz** [comment]
+1

**Comment by vuntz** [comment]
Actually: can you drop the bits for the SLE12 repos? We don't need them anymore.

**Comment by mjura** [comment]
@vuntz : is it correct?

**Comment by vuntz** [comment]
Nope, because now you don't create symlinks for SES 2.1 ;-)

**Comment by mjura** [comment]
@vuntz: As you wish :) I fixed also path to directories

**Comment by vuntz** [comment]
Good catch on the paths! Ideally, this would be a separate commit, though.

+1 after splitting the commit

**Comment by vuntz** [comment]
Actually, I just realized there's still some code later on that is happening for SLE12 only (the checks for the install media). This should be fixed too.

**Comment by vuntz** [comment]
check_or_create_ptf_repository and sign_repositories calls for 12.0 should be dropped too.

**Comment by mjura** [comment]
I discovered also that we have separate check for SLE12 install media, because of ses

**Comment by mjura** [comment]
All changes are in different commits

File: scripts/install-chef-suse.sh [hunk 1]
```diff
@@ -452,95 +452,72 @@ supported_arches_ses="x86_64"
...
-    for arch in $supported_arches; do
-        # Checks for SLE12 SP1 medias
-        MEDIA=/srv/tftpboot/suse-12.1/$arch/install
-
-        # Only x86_64 is truly mandatory; other architectures are only checked
-        # if they exist
-        if [ ! -f $MEDIA/content -a $arch != "x86_64" ]; then
-            continue
-        fi
+for arch in $supported_arches; do
```
Comment (vuntz) [inline scripts/install-chef-suse.sh]: You need something like:
```bash
if is_ses; then
  product_arches="$supported_arches_ses"
else
  product_arches="$supported_arches"
fi

for arch in $supported_arches; do
```

Comment (mjura) [inline scripts/install-chef-suse.sh]: Ahhh, I haven't seen this. You are right. Thank you

File: scripts/install-chef-suse.sh [hunk 2]
```diff
@@ -452,95 +452,78 @@ supported_arches_ses="x86_64"
...
     test ! -e $cloud_dir -a -d $smt_dir && ln -s $smt_dir $cloud_dir
   fi
 done
 
-if ! is_ses; then
-    for arch in $supported_arches; do
-        # Checks for SLE12 SP1 medias
-        MEDIA=/srv/tftpboot/suse-12.1/$arch/install
+if is_ses; then
+  product_arches="$supported_arches_ses"
```
Comment (dirkmueller) [inline scripts/install-chef-suse.sh]: wrong indentation, should be 4 spaces

File: scripts/install-chef-suse.sh [hunk 3]
```diff
@@ -452,95 +452,78 @@ supported_arches_ses="x86_64"
...
+else
+  product_arches="$supported_arches"
```
Comment (dirkmueller) [inline scripts/install-chef-suse.sh]: wrong indentation, should be 4 spaces

File: scripts/install-chef-suse.sh (hunk 2 above)
Comment (mjura) [inline scripts/install-chef-suse.sh]: https://github.com/crowbar/crowbar/pull/2188

File: scripts/install-chef-suse.sh (hunk 3 above)
Comment (mjura) [inline scripts/install-chef-suse.sh]: https://github.com/crowbar/crowbar/pull/2188

**Pull request merged**